package.json

```diff
@@ -15,23 +15,23 @@
     "vue": "^3.3.4"
   },
   "devDependencies": {
-    "@storybook/addon-essentials": "7.1.0-alpha.41",
-    "@storybook/addon-interactions": "7.1.0-alpha.41",
-    "@storybook/addon-links": "7.1.0-alpha.41",
-    "@storybook/blocks": "7.1.0-alpha.41",
-    "@storybook/builder-vite": "7.1.0-alpha.41"
+    "@storybook/addon-essentials": "7.1.0-beta.1",
+    "@storybook/addon-interactions": "7.1.0-beta.1",
+    "@storybook/addon-links": "7.1.0-beta.1",
+    "@storybook/blocks": "7.1.0-beta.1",
     "@storybook/testing-library": "^0.2.0",
-    "@storybook/types": "7.1.0-alpha.41",
+    "@storybook/types": "7.1.0-beta.1",
+    "@storybook/builder-vite": "7.1.0-beta.1",
     "@storybook/vue3": "npm:@storybook-vue/vue3",
     "@storybook/vue3-vite": "npm:@storybook-vue/vue3-vite",
```

Comment:
Hi @floroz you need to resolve to my forked version like this

Reply:
Thank you for taking the time to put together this example.

I have a dumb question because I am perhaps still confused about what I should do.

This repo is a demo project, not just a replacement for the `@storybook/vue3` package. 

At the same time, it's not a published package, and I confess I am not too familiar with overriding an npm dependency directly to a GitHub fork? 

What is the artefact that will be used, since I am not sure if I have to build the project, do a symlink locally, or else.

For now, I just followed your instruction and applied this change:

<img width="785" alt="Screenshot 2023-07-09 at 21 57 08" src="https://github.com/chakAs3/vue3-storybook-custom-render/assets/45793869/9bada725-cba1-4b73-9b5e-fe80a8b43d2b">

Howver, I am getting this error when starting Storybook:

```sh
ReferenceError: defineComponent is not defined
```

And in the console:
<img width="427" alt="Screenshot 2023-07-09 at 22 00 53" src="https://github.com/chakAs3/vue3-storybook-custom-render/assets/45793869/a7ae6671-1c82-4056-a45c-f24f6976e0da">

Is `defineComponent` not supported? For now I still have a mix of that and `script setup` until we've completed our migration.

Again, I appreciate the time to help me!

Reply:
<img width="1121" alt="Screenshot 2023-07-09 at 22 10 05" src="https://github.com/chakAs3/vue3-storybook-custom-render/assets/45793869/a78ad366-9783-407d-bf31-c4bcce1b8cb8">

tried to sync with a pre-release update, although I am pretty sure I am not using the right package

Reply:
can you share your repo ? then i can create a PR for you

Reply:
> can you share your repo? then I can create a PR for you.
> 
> also use the same version of storybook packages as I do. to avoid breaking changes

Regarding dependency, yes first thing I tried was to reset and upgrade to `--prerelease` but I received the same error.

The repo is private unfortunately, would you be able to just explain me what's missing?

```json
    "@storybook/vue3": "npm:@storybook-vue/vue3",
    "@storybook/vue3-vite": "npm:@storybook-vue/vue3-vite",
```

Are these two lines supposed to be exactly like this on my project?